### DISCUSSION TITLE
Getting "unsupported browser" error for my application url in Run mode alone.

### DISCUSSION THREAD

**Discussion by senthiltr** [cypress-io/cypress#17977] [Component Testing]
HI Team,
Uploaded the document with command and result of the command and screenshots.
When I open my application url using chrome and electron with headless mode, i am getting "unsupported browser" error. But when i open other sites it is getting open.
But my application is opening in chrome browser from windows and linux. Only from cypress run mode getting this error.

For my application there is not restriction to use particular version of browser. I am having latest 93 version chrome browser in my system.
Kindly let me know what i need to check?.
[issues.docx](https://github.com/cypress-io/cypress/files/7106245/issues.docx)

**Comment by fabricciotc** [comment]
same error
